### ISSUE TITLE
119.0.6045.214 AVX breaks passwords and more

### ISSUE THREAD

**Issue by MaggotHATE** [Alex313031/thorium-win7#38]
**System Details**
 - OS: Windows 8.1
 - Thorium Version AVX 119.0.6045.214

**Problem**
Updating from 109.0.5414.173 to 119.0.6045.214 (AVX, always zips) broke my passwords, making them into gibberish. When I tried rolling back to 109.0.5414.173 (replaced back "chrome_proxy.exe", "initial_preferences", "thor_ver" and "thorium.exe", "109.0.5414.173" folder is in BIN too), it broke the entire browser and my userdata (- 2 months of...everything, I guess?). Now I have to use my ~old userdata backup and 109.0.5414.173. 

**Additional Notes**
I expected this release to be experimental, of course, but not that destructive.

**Comment by Alex313031** [comment]
@MaggotHATE Yeah.... That's what can happen when you upgrade or downgrade 10 versions. There's no way to fix it AFAIK. Chromium only guarantees user profile backwards/forwards compatibility up to two major versions.

**Comment by MaggotHATE** [comment]
@Alex313031 So, what can I do to have a chance of updating safely in such case? Or am I stuck on 109? Does `--disable-encryption` have any effect on that?

**Comment by Templayer** [comment]
> @Alex313031 So, what can I do to have a chance of updating safely in such case? Or am I stuck on 109? Does `--disable-encryption` have any effect on that?

Personally, I back up both the browser and my personal data (USER_DATA in the portable version) before updating, and storing passwords in the browser itself is bad juju. I'm using the IronVest extension for that instead! And I also have a central place where I put all of my passwords, just in case of IronVest screwing it up...

**Comment by Alex313031** [comment]
@MaggotHATE I would reccomend just deleting all passwords and re-log into your sites, or start with a fresh profile on M122 (latest release), and re-login to your sites there. You could back up your bookmarks.

**Comment by MaggotHATE** [comment]
@Alex313031 Well, thanks for answering! Guess it's time to get more used to Bitwarden - with how things are going I'll have to move to Ubuntu anyway. It was a good run on win8/8.1, time to move on.

**Comment by Templayer** [comment]
> @Alex313031 Well, thanks for answering! Guess it's time to get more used to Bitwarden - with how things are going I'll have to move to Ubuntu anyway. It was a good run on win8/8.1, time to move on.

Yeah, I've spent over three years in Linux Mint (which is an Ubuntu derivative... Cinnamon is just more Windows-like for me, as far as system GUIs go) on my unofficial work laptop. But I'm still on Win7... and WinXP on my main desktop, as that is where I do most of my gaming. There are games I installed over 20 years ago now (2003), and since I have been moving that operating system from drive to drive, they still work. :D :D
